JULIERAJ/KIDS-FIRST-Front
# Fix: edit kid photo dropdown

One Line Description
Edit kid photo dropdown style

Requirements
- Expectation: Please refer to [Figma ](https://www.figma.com/design/qUBlZE5TIgpqShKTVcSvRo/Kids-First-WITT(Desktop)?node-id=14512-130900&t=seuHdaHYobfLzmRZ-4)design.
- NOTE : There is no Figma design for effect of mousing over dropdown items. Check with designers on expectation.

Notes
There is a Figma design illustrating the effect of mousing over dropdown items, which can be found here:
https://www.figma.com/proto/qUBlZE5TIgpqShKTVcSvRo/Kids-First-WITT(Desktop)?node-id=14512-130520&node-type=frame&t=iItelqg7RezdCKWy-0&scaling=scale-down&content-scaling=fixed&page-id=0%3A1&starting-point-node-id=14909%3A60377

Test Steps
1. Open the application. 
2. Navigate to the kid's info page
3. Check Edit Kid Photo dropdown design 

UI/UX Outcome
![image](https://github.com/user-attachments/assets/675c7418-c4f0-4207-a515-4b8f0e9cf0dc)

Checklist
